tests/js/spec/components/group/externalIssueForm.spec.jsx

```diff
@@ -48,4 +54,49 @@ describe('SentryAppExternalIssueForm', () => {
       expect(wrapper.find('Form').prop('apiMethod')).toEqual('POST');
     });
   });
+
+  describe('link', () => {
+    beforeEach(() => {
+      wrapper = mount(
+        <SentryAppExternalIssueForm
+          group={group}
+          sentryAppInstallation={sentryAppInstallation}
+          config={component.schema}
+          action="link"
+        />,
+        TestStubs.routerContext()
+      );
+    });
+
+    it('specifies the action', () => {
+      expect(wrapper.find('Form').prop('initialData').action).toEqual('link');
+    });
+
+    it('specifies the group', () => {
+      expect(wrapper.find('Form').prop('initialData').groupId).toEqual(group.id);
+    });
+
+    it('renders each required_fields field', () => {
+      component.schema.link.required_fields.forEach(field => {
+        expect(wrapper.exists(`#${field.name}`)).toBe(true);
+      });
+    });
+
+    it('renders prepopulated defaults', () => {
+      expect(wrapper.find('Input #a').prop('value')).toEqual(`${group.title}`);
+      const description = `Sentry Issue: [${group.shortId}](${group.permalink}?referrer=${sentryApp.name})`;
+      expect(
+        wrapper
+          .find('TextArea #c')
```

Comment:
What's with the `#c` and `#a` selectors here? Where are you getting those IDs from?

Reply:
it's basically what you are doing here but with the specific ids for the field that has the default https://github.com/getsentry/sentry/blob/master/tests/js/spec/components/group/externalIssueForm.spec.jsx#L41

Reply:
oh I see. What if we just pulled these into variables named `issueTitleField` and `issueDescriptionField` or something so it's a little more obvious? The random single letter IDs were pretty mysterious.